src/main.rs

```diff
@@ -1,34 +1,18 @@
-use action_validator::Config;
-use clap::Parser;
-use std::process;
+#[cfg(feature = "js")]
+fn main() {}
 
+#[cfg(not(feature = "js"))]
 fn main() {
-    let config = Config::parse();
+    use action_validator::CliConfig;
+    use clap::Parser;
+    use std::process;
 
-    // Run the validator on each file and collect any errors
```

Comment:
I can't seem to find where this code has gotten to.  I suspect your merge conflict resolution might have nuked it.

Reply:
Yeah the merge conflicts were pretty bad - there was basically no overlap 😬 this code has moved to `cli::run` but looks a bit different now. Functionality is the same as far as I can tell from a few manual tests. I can refactor to make it more similar if that's preferable and/or add some automated tests for the new functionality?

Reply:
I really should have put automated tests in when #31 was merged, shouldn't I... I'll add one to `main` now, for cleanliness, and then if your PR is still clean against that, I think we're ready for merge.

Reply:
Exellent! Thanks for picking that up =) Let me know if there's anything I can do to help